# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xli/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

No cargs.h anymore ...

  - Rene Rebe <rene@exactcode.de>

--- xli-1.17.0/rlelib.c.vanilla	2005-05-30 16:06:12.000000000 +0200
+++ xli-1.17.0/rlelib.c	2005-05-30 16:06:24.000000000 +0200
@@ -12,7 +12,7 @@
  */
 
 #include "xli.h"
-#include <varargs.h>
+#include <stdarg.h>
 #include <ctype.h>
 #include "rle.h"
 
